app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :user_signup_params, :resource_name
+
+  def resource_name
```

Comment:
What happens if this is not defined/overwritten?

Reply:
actually, it still works, so I removed it